hack/release.sh

```diff
@@ -26,7 +26,7 @@ set -x
 DO_WINDOWS_BUILD=${DO_WINDOWS_BUILD_ENV:-true}
 
 # BASE_REPO is the root path of the image repository
-readonly BASE_IMAGE_REPO=us-central1-docker.pkg.dev/k8s-staging-images/csi-vsphere
```

Comment:
we don't have either
us-central1-docker.pkg.dev/k8s-staging-images/csi-vsphere or
registry.k8s.io/csi-vsphere